CONTRIBUTING.md

```diff
@@ -1,5 +1,11 @@
 # How to Contribute
 
+## To Cilium/Hubble
+
 See the [Developer / Contributor
 Guide](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/) for detailed information on
 how to contribute, get started and find good first issues.
+
+## To the website
+
+Please see the [website contributing guide](https://github.com/cilium/cilium.io/CONTRIBUTING.md) for detailed information on how to add blogs, trainings, and other resources
```

Comment:
for when after is https://github.com/cilium/cilium.io/pull/11 merged

Should the link be https://github.com/cilium/cilium.io/blob/main/CONTRIBUTING.md istead ?